server/container/guice/cassandra-rabbitmq-guice/src/main/java/org/apache/james/modules/blobstore/BlobStoreChoosingConfiguration.java

```diff
@@ -33,7 +33,7 @@
 
     public enum BlobStoreImplName {
         CASSANDRA("cassandra"),
-        OBJECTSTORAGE("objectstorage"),
+        S3("s3"),
```

Comment:
This braking changes needs to be documented.

Please also don't forget to update default configuration.

Reply:
why not keeping as it is? we just use the S3 client and the service behind doesn't change (Swift)... This change make me feel like we're allowing S3 only for blobStore

Reply:
We do use only S3. And Swift has a S3 API.
It's much more usual to use "S3" wording than "objectstorage" so it may help the users to talk about S3.